README.md

```diff
@@ -62,6 +62,19 @@ To deploy a production release:
     3. Type in the tag name you want to release for example `v1.2.3`.  (Note this won't work until the PR has been merged to master)
     4. Click the `Run Workflow` button.
 
+### Testing in CLUE
+
+The diagram-view is used in CLUE in the diagram tile, and it can be useful to test changes made in Quantity Playground in CLUE without deploying updates. To do so, it's best to use `yalc`.
+
+1. Install Yalc by running `sudo npm install --global yalc`.
+2. Install Rollup by running `sudo npm install --global rollup`.
```

Comment:
I generally don't install `yalc` globally, preferring to just use `npx yalc ...` as needed. For tools like `rollup`, generally local install is preferred. Is there a reason you're recommending global install?

Reply:
I was following the advice from these sites:
https://github.com/wclr/yalc
https://rollupjs.org/guide/en/

Installing globally is how I was able to get this to work, so that's what I included in my writeup. However I don't have strong feelings about it and will change these lines if you think there's a better way to do it.

Reply:
For libraries like `webpack` and `rollup`, it is common for different versions to be used in different projects, so local installation is generally preferred. For `yalc`, it's not actually a dependency of the project so the same concern doesn't apply, but personally I still try to avoid global npm installs and so prefer to use `npx yalc ...` instead.

Reply:
Alright that's fine with me. What should I replace the lines I wrote with? And will someone have to install yalc separately in both projects using npx?

Reply:
The `rollup` dependency should be listed in the `package.json` with the other dependencies and should therefore be installed via `npm install` along with the other dependencies. As for `yalc`, `npx` supports executing packages without installing them, so no installation of `yalc` (global or local) should be necessary to run `npx yalc ...`. In short, neither of these lines should be necessary.